front-office-integration-examples/dispatch-control-example/02-DocumentAddingWatermark/src/index.ts

```diff
@@ -0,0 +1,106 @@
+import { createCommands } from "./commandsCreator";
+import { getCommandFileName } from "../../Utils/commandUtils";
+import { Messages } from "../../Utils/messages";
+import {
+  pathCombine,
+  DocumentsForWatermarkPath,
+  DocumentsWithWatermarkPath,
+} from "../../Utils/pathUtils";
+
+export function getDescription(): ScriptDescription {
+  return {
+    description: "Add watermark to documents.",
+    input: [
+      {
+        id: "workingDirectory",
+        displayName: "WorkingDirectory",
+        description: "WorkingDirectory",
+        type: "Connector",
+        defaultValue: "blob://FoIntegration",
+        required: true,
+        readonly: true,
+      },
+      {
+        id: "watermarkPath",
+        displayName: "watermarkPath",
+        description: "Path to file with content to add (Blob storage).",
```

Comment:
Jestli cesta ma jit do nejakeho souboru, ktery cte IPS a kvuli tomu tam mas to omezeni na blob storage, tak lepsi cesta je neomezovat vstup, soubor prekopirovat na blob a do commandu pro IPS dat cestu na blob

Reply:
pokud bys chtel, muzes kontrolovat, jestli je to job, blob, map/data, share, a nechat to tam, jinak prekopirovat na blob.. Ale uplne bych ten vstup neomezoval.

Reply:
Zkopírovat na blob a neomezovat vstup